### PR TITLE
corosync: bump to 2.4.5

### PR DESCRIPTION
**Pull request by grueni**

### CODE REVIEW

File: components/cluster/corosync/patches/man8.patch [hunk 1]
```diff
@@ -331,6 +331,15 @@ diff -Nur a/man/corosync-qdevice-net-certutil.8 b/man/corosync-qdevice-net-certu
  .SH NAME
  corosync-qdevice-net-certutil - tool to generate qdevice model net TLS certificates
  .SH SYNOPSIS
+@@ -70,7 +70,7 @@
+ and
+ .B corosync-qdevice
+ certificates on all nodes. It's highly recommended that you use an ssh agent,
+-or ssh/scp will keep asking for a password - roughly 8 times the number of nodes.
++or ssh/scp will keep asking for a password - roughly 1m times the number of nodes.
```
Comment (Mno-hime) [inline components/cluster/corosync/patches/man8.patch]: This seems to be unintended change.

File: components/cluster/corosync/patches/man8.patch [hunk 2]
```diff
@@ -1077,9 +1086,58 @@ diff -Nur a/man/sam_overview.8 b/man/sam_overview.8
...
-+++ b/man/votequorum.5	2017-09-26 20:25:14.945631782 +0000
+diff -Nur b/man/votequorum.5 a/man/votequorum.5
+--- b/man/votequorum.5	2019-08-19 07:35:39.871503472 +0000
++++ a/man/votequorum.5	2019-08-19 07:47:38.778202474 +0000
+@@ -61,7 +61,7 @@
+ both will disable votequorum. If both are present at the same time, 
+ the quorum.expected_votes value will override the one calculated from the nodelist.
+ .PP
+-Example (no nodelist) of an 8 node cluster (each node has 1 vote):
++Example (no nodelist) of an 1m node cluster (each node has 1 vote):
```
Comment (Mno-hime) [inline components/cluster/corosync/patches/man8.patch]: Same thing here. And several other places.

Comment (grueni) [reply components/cluster/corosync/patches/man8.patch]: Yes, too much gsed

**Comment by alarcher** [comment]
@grueni @Mno-hime Any inter-dependencies between the coming PRs that I should be aware of?

**Comment by grueni** [comment]
Order of merging should be
- libqb
- resource-agents
- corosync (needs libqb)
- pacemaker (needs corosync, libqb, resource-agents)
- crmsh (needs pacemaker, corosync)
- pcs (needs pacemaker, corosync)

I checked it with snmp-5.7.3 and snmp-5.8.
And if possible have a look at the PR with net-snmp 5.8 https://github.com/OpenIndiana/oi-userland/pull/5168.

**Review by Mno-hime — APPROVED** [review]
PR looks good, provided it works fine with pacemaker.

**Pull request merged**